deepchecks/vision/suites/default_suites.py

```diff
@@ -25,9 +28,87 @@
 __all__ = ['train_test_validation', 'model_evaluation', 'full_suite', 'integrity_validation', 'data_integrity']
 
 
-def train_test_validation(**kwargs) -> Suite:
-    """Create a suite that is meant to validate correctness of train-test split, including integrity, \
-    distribution and leakage checks."""
+def train_test_validation(n_top_show: int = 5,
+                          label_properties: List[Dict[str, Any]] = None,
+                          image_properties: List[Dict[str, Any]] = None,
+                          sample_size: int = 10_000,
+                          random_state: int = None,
+                          **kwargs) -> Suite:
+    """Suite for validating correctness of train-test split, including integrity, \
+    distribution and leakage checks.
+
+    List of Checks:
+        .. list-table:: List of Checks
+           :widths: 50 50
+           :header-rows: 1
+
+           * - Check Example
+             - API Reference
+           * - :ref:`plot_vision_new_labels`
+             - :class:`~deepchecks.vision.checks.train_test_validation.NewLabels`
+           * - :ref:`plot_vision_similar_image_leakage`
+             - :class:`~deepchecks.vision.checks.train_test_validation.SimilarImageLeakage`
+           * - :ref:`plot_vision_heatmap_comparison`
+             - :class:`~deepchecks.vision.checks.train_test_validation.HeatmapComparison`
+           * - :ref:`plot_vision_train_test_label_drift`
+             - :class:`~deepchecks.vision.checks.train_test_validation.TrainTestLabelDrift`
+           * - :ref:`plot_vision_image_property_drift`
+             - :class:`~deepchecks.vision.checks.train_test_validation.ImagePropertyDrift`
+           * - :ref:`plot_vision_image_dataset_drift`
+             - :class:`~deepchecks.vision.checks.train_test_validation.ImageDatasetDrift`
+           * - :ref:`plot_vision_feature_label_correlation_change`
+             - :class:`~deepchecks.vision.checks.train_test_validation.FeatureLabelCorrelationChange`
+
+    Parameters
+    ----------
+    n_top_show: int, default: 5
+        Number of images to show, sorted by the similarity score between them.
```

Comment:
1. Sorted by the similarity score? (or... each check that has this parameter - by it's own criteria?)
2. (separate issue) -> why aren't we consistent between tabular and vision api? (we should be...)

Reply:
1. done
2. It's a bigger problem we're not gonna solve in this PR. I agree we should open an issue for that...

Reply:
For 2 - yes, for sure
Wrote it hoping it will make the issue magically appear since you are a magician! anyway I'll open it now

Reply:
Opened #1697